osu.Game/Screens/PlayfieldBorder.cs

```diff
@@ -6,14 +6,14 @@
 using osu.Framework.Graphics.Shapes;
 using osuTK.Graphics;
 
-namespace osu.Game.Screens.Edit.Compose.Components
+namespace osu.Game.Screens
```

Comment:
This is not a good namespace for this component. I would suggest

```suggestion
namespace osu.Game.Rulesets.UI
```